vllm/model_executor/sampling_metadata.py

```diff
@@ -549,7 +546,7 @@ def from_lists(cls, temperatures: List[float], top_ps: List[float],
             device="cpu",
             dtype=torch.long,
             pin_memory=pin_memory,
-        ).T.contiguous()
+        ).t().contiguous()
```

Comment:
This change is to avoid pytorch warning because `sampling_seeds_t` becomes 1-dim here.